src/main/kotlin/team/themoment/gsmNetworking/domain/mentor/service/QueryTempMentorListService.kt

```diff
@@ -31,8 +31,9 @@ class QueryTempMentorListService (
                 TempMentorInfoDto.CompanyInfoDto(
                     tempMentor.companyName,
                     tempMentor.companyUrl),
```

Comment:
```suggestion
                TempMentorInfoDto.CompanyInfoDto(
                    tempMentor.companyName,
                    tempMentor.companyUrl
                ),
```
이런식으로 마지막에 한칸 띄우는건 어떤가요? `option + command + l `누르시면 자동완성 되는걸 사용하는걸 추천해요.